src/components/Card/Card.js

```diff
@@ -0,0 +1,33 @@
+import * as React from 'react';
+import * as PropTypes from 'prop-types';
+import classNames from 'classnames';
+import getMergedClassNames from '../../utils/getMergedClassNames';
+import styles from './style.scss';
+
+const baseClass = 'card';
+
+const Card = ({ img, title, children }) => {
+  const hasComponents = React.Children.count(children) >= 1;
+  const mergedClassNames = getMergedClassNames(styles[baseClass]);
+  const titleClass = classNames({
+    [`${baseClass}__title`]: hasComponents,
+    [`${baseClass}__empty`]: !hasComponents
```

Comment:
can you describe any use case for this empty Card?

Reply:
So, this case occurs in customer details, if someone doesn't provide content for the card.

Reply:
I designer is ok with that... :P

Reply:
This comes from the customer details section 🤔 maybe they don't love it, but should we change it now?

Reply:
IMHO there should be  always an empty state when content is not provided, but we can't force it

Reply:
What do you suggest?